src/styles/main.scss

```diff
@@ -1,3 +1,71 @@
+$month-lengths: 28, 29, 30, 31;
+$cell-size: 100px;
+$days: (
+  "mon" : 1,
+  "tue" : 2,
+  "wen" : 3,
+  "thu" : 4,
+  "fri" : 5,
+  "sat" : 6,
+  "sun" : 7
+);
+
 body {
   margin: 0;
 }
+
+.calendar__content {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $cell-size * 7 + 7;
+  margin: 0 auto;
+}
+
+.calendar {
+  padding: 20px;
+  $root: &;
```

Comment:
Per checklist:
> [BEM] - Create a separate file per each BEM block styles that have the same name as the block
Make sure to repeat the topics on BEM if you need to

Also it makes sense to have separate files for variables and a dedicated main file.